apps/researcher/src/app/[locale]/objects/[id]/(provenance)/group-events.test.ts

```diff
@@ -2,7 +2,20 @@ import {describe, expect} from '@jest/globals';
 import {groupByDateRange} from './group-events';
 import {LabeledProvenanceEvent} from './definitions';
 
-const locale = 'en';
+// Is simple `formatDateRange` mock that returns a string representation of a date range.
+const formatDateRange = ({
```

Comment:
For consistency with other parts of the codebase, use `function formatDateRange()`?